modules/core/src/main/java/org/apache/ignite/internal/util/OffheapReadWriteLock.java

```diff
@@ -33,6 +33,9 @@
  * </pre>
  */
 public class OffheapReadWriteLock {
+    /** Default concurrency level for the lock. */
+    public static final int DEFAULT_CONCURRENCY_LEVEL = 128;
```

Comment:
Could you describe why exactly 128 and not 64 or 256?

Reply:
I don't know, I extracted what was there already. This explanation has nothing to do with my PR

Reply:
Sad Ok